### PR TITLE
 Translate to Simplified Chinese: Optimized for Change

### PR DESCRIPTION
**Pull request by HanchengZhao**

### CODE REVIEW

**Comment by chloewlin** [comment]
Hi @HanchengZhao , thank you for submitting this translation! So @boomyao also submitted a translation on #304 . Do you mind working with @boomyao to see how you can collaborate and submit just one version for this article? 🙂

Some of my thoughts. There is no right or wrong. Just different styles. 
- I prefer your `起因和结果` for `cause and effect` on line 19.
- I prefer @boomyao 's `应变而优` for `optimized for change` on line 23.

File: src/pages/optimized-for-change/index.zh-hans.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: 为迎接改变优化
+date: '2018-12-12'
+spoiler: 什么造就了优秀的API?
+---
+
+什么造就了优秀的API？
+
+*优秀的* API 设计是便于记忆以及清晰不含糊的。它能促使代码变得可读性强，执行正确，以及性能优异, 也能帮助开发者进入[成功之坑](https://blog.codinghorror.com/falling-into-the-pit-of-success/)。
+
+我把这些设计层面叫做 “第一级”，因为这些是一个库开发者需要关注的优先事项。你可能需要在它们中的某些上做妥协和权衡，但是至少这些都是你的考虑点。
```
Comment (chloewlin) [inline src/pages/optimized-for-change/index.zh-hans.md]:
```suggestion
我把这些设计层面叫做 「第一级」，因为这些是一个库开发者需要关注的优先事项。你可能需要在它们中的某些上做妥协和权衡，但是至少这些都是你的考虑点。
```
This is just my suggestion. Using square brackets will increase readability. Also see reference [here](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-CN.md#%E7%AE%80%E4%BD%93%E4%B8%AD%E6%96%87%E4%BD%BF%E7%94%A8%E7%9B%B4%E8%A7%92%E5%BC%95%E5%8F%B7).

File: src/pages/optimized-for-change/index.zh-hans.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+然而，除非你是要将一个漫游者发往火星，不然你的代码很有可能随着时间而改变。同样，你API使用者的代码也是一样。
```
Comment (chloewlin) [inline src/pages/optimized-for-change/index.zh-hans.md]:
```suggestion
然而，除非你是要将一个漫游者发往火星，不然你的代码很有可能随着时间而改变。同样，你 API 使用者的代码也是一样。
```
Add whitespaces between English and Chinese characters. See reference [here](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-CN.md#%E4%B8%AD%E8%8B%B1%E6%96%87%E4%B9%8B%E9%97%B4%E9%9C%80%E8%A6%81%E5%A2%9E%E5%8A%A0%E7%A9%BA%E6%A0%BC).

Comment (HanchengZhao) [reply src/pages/optimized-for-change/index.zh-hans.md]: Hi Chloe, thanks for reviewing! I'll talk to @boomyao to see what we can do to make this better.

**Comment by HanchengZhao** [comment]
@chloewlin @boomyao I have just updated my translation by taking the good parts of both versions. It's more about preference, no right or wrong here. Let me know if there's anything you want to make a change. Thank you guys for the help and the work!

**Comment by chloewlin** [comment]
@HanchengZhao good job collaborating with @boomyao. I like this translation a lot. I am merging this in now. 🥇Thank you so much for your contribution! 🙂

**Pull request merged**